### PR TITLE
Add types for meteor/ostrio:files

### PR DESCRIPTION
**Pull request by OliverColeman**

### CODE REVIEW

File: types/meteor-files/tslint.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ 
+  "extends": "dtslint/dt.json",
+  "rules": {
```
Comment (sheetalkamat) [inline types/meteor-files/tslint.json:3]: This rule needs to be removed and the error needs to be fixed

File: types/meteor-files/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts"
```
Comment (sheetalkamat) [inline types/meteor-files/tsconfig.json:21]: Need test

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/meteor-files/tslint.json (hunk 1 above)
Comment (OliverColeman) [reply types/meteor-files/tslint.json:3]: The package is not in npm. It is a Meteor js package, hosted here https://atmospherejs.com/ostrio/files

Comment (sheetalkamat) [reply types/meteor-files/tslint.json:3]: When you remove it , lint will show you how to fix the error

File: types/meteor-files/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+// Minimum TypeScript Version: 3.7
+
+/// <reference types="meteor" />
+
+// tslint:disable-next-line no-single-declare-module
+declare module "meteor/ostrio:files" {
+  import { Mongo } from 'meteor/mongo';
+  import { ReactiveVar } from 'meteor/reactive-var';
+  import { SimpleSchemaDefinition } from 'simpl-schema';
+  import * as http from "http";
```
Comment (amcasey) [inline types/meteor-files/index.d.ts:14]: Nit: inconsistent quotes

**Review by amcasey — CHANGES_REQUESTED** [review]
I think I'm missing something - it looks like the package provides types directly? https://github.com/VeliovGroup/Meteor-Files/blob/master/index.d.ts

**Comment by OliverColeman** [comment]
Ahh it does provide them, with a (very poor) version I posted a while ago! I'll confer with the package maintainer. Thanks for looking at this! :)

**Pull request closed without merging**